frontend/src/components/intro/Introduction/Introduction.js

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback } from 'react';
+import ButtonMaterial from 'components/common/Button';
+import { withRouter } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import './Introduction.css';
+
+function Introduction({history}) {
+  
+  const onStart = useCallback(
+    () => {
+      history.push("/request");
+    },
+    [history],
+  );
+
```

Comment:
Please please this logic into `IntroductionContainer` and pass `onStart` function to `Introduction` component as props